packages/edit-site/src/components/page-patterns/index.js

```diff
@@ -78,6 +78,7 @@ export default function DataviewsPatterns() {
 	const categoryId = categoryIdFromURL || PATTERN_DEFAULT_CATEGORY;
 	const [ view, setView ] = useState( DEFAULT_VIEW );
 	const previousCategoryId = usePrevious( categoryId );
+	const previouspostType = usePrevious( type );
```

Comment:
It'd be good to fix the capitalization of the variable - `previousPostType`

Reply:
Thanks @ramonjd for fixing variable name.